.github/workflows/publish-sdk.yaml

```diff
@@ -6,35 +6,50 @@ on:
 
 env:
   ANDROID_API_LEVEL: 33
-  ORG_GRADLE_PROJECT_mavenCentralUsername: ${{ secrets.OSSRH_USERNAME }}
-  ORG_GRADLE_PROJECT_mavenCentralPassword: ${{ secrets.OSSRH_PASSWORD }}
   CI: true
 
 jobs:
   test:
     uses: ./.github/workflows/test.yaml
   publish:
+    env:
+      ORG_GRADLE_PROJECT_mavenCentralUsername: ${{ secrets.MAVEN_CENTRAL_TOKEN_USERNAME }}
+      ORG_GRADLE_PROJECT_mavenCentralPassword: ${{ secrets.MAVEN_CENTRAL_TOKEN_PASSWORD }}
+      ORG_GRADLE_PROJECT_signingInMemoryKey: ${{ secrets.GPG_PRIVATE_KEY }}
+      ORG_GRADLE_PROJECT_signingInMemoryKeyPassword: ${{ secrets.GPG_PASSPHRASE }}
+      GPG_PRIVATE_KEY: ${{ secrets.GPG_PRIVATE_KEY }}
+      GPG_PASSPHRASE: ${{ secrets.GPG_PASSPHRASE }}
     needs: test
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
 
       - name: Set up JDK 17
         uses: actions/setup-java@v4
         with:
           java-version: 17
           distribution: 'adopt'
-          gpg-private-key: ${{ secrets.GPG_PRIVATE_KEY }}
-          gpg-passphrase: ${{ secrets.GPG_PASSPHRASE }}
 
       - name: Restore gradle.properties
         shell: bash
         run: |
           mkdir -p ~/.gradle/
           echo "GRADLE_USER_HOME=${HOME}/.gradle" >> $GITHUB_ENV
+      
+      - name: Publish and Release to Maven Central Portal
+        working-directory: eppo
```

Comment:
Vanniktech plugin has trouble finding the artifacts since the project is nested under `eppo`. Adjusting the working dir seems to be the best way to point it at the right base dir

Reply:
Ah yeah I encountered this issue when manually running gradle commands. Did you try `:eppo:` prefix for gradle command?
```
./gradlew :eppo:publishReleasePublicationToMavenLocal
```